bin/templates/scripts/cordova/lib/projectFile.js

```diff
@@ -40,7 +39,7 @@ function parseProjectFile (locations) {
     xcodeproj.parseSync();
 
     const xcBuildConfiguration = xcodeproj.pbxXCBuildConfigurationSection();
-    const plist_file_entry = _.find(xcBuildConfiguration, entry => entry.buildSettings && entry.buildSettings.INFOPLIST_FILE);
+    const plist_file_entry = Object.values(xcBuildConfiguration || {}).find(entry => entry.buildSettings && entry.buildSettings.INFOPLIST_FILE);
```

Comment:
We could probably reduce this to the following:

```suggestion
    const plist_file_entry = Object.values(xcBuildConfiguration).find(entry => entry.buildSettings && entry.buildSettings.INFOPLIST_FILE);
```

Since in the next line, `plist_file_entry` is assumed to be truthy, which cannot be the case if `xcBuildConfiguration` is falsy.